ci-operator/config/openstack-k8s-operators/edpm-ansible/OWNERS

```diff
@@ -0,0 +1,15 @@
+approvers:
+  - cescgina
+  - cjeanner
+  - rlandy
+  - raukadah
+  - sandeepyadav93
+  - viroel
```

Comment:
I would suggest to add following members to the list(based on (https://github.com/openstack-k8s-operators/edpm-ansible/graphs/contributors)
- slagle
- rebtoor
- kajinamit
- bshephar
- raukadah
- rabi

Reply:
Thank you, I've updated it.